languages/julia/config.json

```diff
@@ -8,6 +8,46 @@
         "indent_style": "space",
         "indent_size": 4
     },
+    "tags": [
+        "typing/dynamic",
+        "paradigm/declarative",
+        "paradigm/functional",
+        "paradigm/imperative",
+        "paradigm/object_oriented",
+        "paradigm/procedural"
+    ],
+    "key_features": [
+        {
+            "title": "Fast",
+            "content": "Julia is designed for high performance. Code compiles to efficient native code for various platforms",
+            "icon": ""
+        },
+        {
+            "title": "Interactive",
+            "content": "Julia is dynamically typed, feels like a scripting language and has good support for interactive use",
+            "icon": ""
+        },
+        {
+            "title": "Scientific",
+            "content": "Julia is designed with reproducibility, distributed computing and easy collaboration in mind",
+            "icon": ""
+        },
+        {
+            "title": "Composable",
+            "content": "Julia uses multiple dispatch as a paradigm, making it unreasonably effective at composing programs",
+            "icon": ""
+        },
+        {
+            "title": "Expressive",
+            "content": "Julia's familiar mathematical syntax and metaprogramming features lead to highly expressive code",
+            "icon": ""
+        },
+        {
+            "title": "General",
+            "content": "",
```

Comment:
Not sure what to put here. The website has 

> Julia provides asynchronous I/O, metaprogramming, debugging, logging, profiling, a package manager, and more. One can build entire Applications and Microservices in Julia.

which I don't find particularly compelling to point out that it's indeed capable of doing more than technical computing.